.licenserc.yaml

```diff
@@ -57,6 +57,7 @@ header:
     - '**/src/main/proto/jaeger/**'
     - '**/src/main/proto/mixer/**'
     - '**/src/main/proto/policy/**'
+    - '**/src/main/proto/satellite/**'
```

Comment:
I'd say these files are our own work, not third parties', so please add license headers to them instead of excluding them

Reply:
I was struggling about this, right now, I think yes, this is written and provided by us, let's add license for it.